src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -1060,4 +1060,10 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Hyperlink_Owner" xml:space="preserve">
     <value>Owner</value>
   </data>
+  <data name="ToolTip_UseDotnetCommand" xml:space="preserve">
+    <value>To know this package transitive origin path use command</value>
+  </data>
+  <data name="ToolTip_DotnetNuGetWhy" xml:space="preserve">
+    <value>dotnet nuget why</value>
```

Comment:
I'm worried this gets accidentally localized.
It should be done differently.

Reply:
Yeah I was wondering the same thing, I'm just going to "hard code" the string in the XAML